webpack/webpack.common.js

```diff
@@ -314,6 +322,13 @@ module.exports = () => {
                     jQuery: Path.resolve(__dirname, PROJECT_RELATIVE_NODE_MODULES_PATH, 'jquery', 'dist', 'jquery.min'),
                     jquery: Path.resolve(__dirname, PROJECT_RELATIVE_NODE_MODULES_PATH, 'jquery', 'dist', 'jquery.min')
                 }),
+                ...Object.keys(entryPoints).map(name => {
```

Comment:
This is the one bit of fancy footwork in this PR. This essentially iterates all entry points and adds `media/js/template-loader.js` as a requirement. This is necessary for the KOJS override to happen in every component.